docs/tutorial-2.md

```diff
@@ -0,0 +1,39 @@
+# How to start an Action
+The last chapter explained how to create a Cicero Action.
+
+Now it makes sense explain how to actually start this Action.
+
+## Enter the Cicero development shell
+
+```
+nix develop
+```
+
+## Create json representation of the fact
+
+A fact can have multiple inputs for providing different arguments for calling the actual Action.
+
+But in fact the examples/runners/bash doesn't really need any of those inputs for the moment.
+```
+cat > /tmp/bash-fact.json << EOF
+{
+}
+EOF
+```
+
+## Use httpie to create a new Fact
+The creation or update of Facts will trigger the corresponding Actions to run in Cicero.
+
+```
+http -v post :8080/api/fact "examples/runners/bash":=@/tmp/bash-fact.json
+```
```

Comment:
I understand point of creating description of the fact in file.

However for me given command did not work (wrong port). And I was not able to figure out what are we doing in here (creating fiels using cat, http that has POST as lower case.

I would prefer information, that we are creating a file that will be used to create JSON request:
```json
{
    "examples/runners/bash": {}
}
```

It is obvious *after* you do it successfully and understand what is purpose of cicero.

Maybe we would like to have 2 examples here:
one that just trigger run without a file:
```bash
http -v POST :8000/api/fact examples/runners/bash:={}
```
and other (with different action?) that use file but have something interesting configured.

WDYT?

Reply:
The command should be fixed by now.
Also added a better description.